app/models/survey/poll.rb

```diff
@@ -4,7 +4,7 @@ class Survey::Poll < ApplicationRecord
   include FilterByRole
 
   has_one :date, as: :dateable, class_name: "FixedDate", dependent: :destroy
-  has_many :questions, class_name: "Survey::Question", foreign_key: "survey_poll_id", dependent: :restrict_with_error
+  has_many :questions, class_name: "Survey::Question", foreign_key: "survey_poll_id", dependent: :destroy
```

Comment:
mit `:restrict_with_error` wurde folgendes gemeldet:

<img width="1216" alt="Bildschirmfoto 2021-06-17 um 17 48 08" src="https://user-images.githubusercontent.com/1942953/122435495-5603cb80-cf98-11eb-980b-d27043bab3c5.png">

das vorhandensein einer `Question` is doch aber nicht ausschlaggebend dafür, dass etwas nicht gelöscht werden kann. jede Umfrage hat eine Question und ebenfalls mind. 2 und max. 10 `ResponseOption`s. Somit ist dort ebenfalls das `:restrict_with_error` entfernt. Das vorhandensein von antwortmöglichkeiten sollte nicht das löschen einer umfrage verhindern, da es immer welche gibt.

Reply:
ja, nur wird vermutlich so der error von ResponseOptions nicht mehr nach oben zu den Polls, transportiert. ich teste das nochmal

Reply:
Nach oben zu den Polls muss es aber gar nicht denke ich, da ja alles gelöscht werden soll. Beim Löschen einer einzelnen Antwort soll es eine Prüfung auf Stimmen geben, da muss dann aber nichts "hoch zu Polls", oder seh ich was falsch?